pulseapi/users/test_views.py

```diff
@@ -0,0 +1,43 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from pulseapi.users.serializers import EmailUserSerializer
+from pulseapi.users.test_models import EmailUserFactory
+
+
+class TestUserView(TestCase):
+    def setUp(self):
+        self.users = [EmailUserFactory() for i in range(2)]
+        for user in self.users:
+            user.save()
+
+    # def test_list_users_returns_user_data(self):
```

Comment:
I'm not sure where these came from, but yeah I don't think they're useful. Removing.